css/filter-effects/inheritance.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Inheritance of Filter Effects properties</title>
+<link rel="help" href="https://drafts.fxtf.org/filter-effects/#property-index">
+<meta name="assert" content="Properties inherit according to the spec.">
+<meta name="assert" content="Properties have initial values according to the spec.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css/support/inheritance-testcommon.js"></script>
+</head>
+<body>
+<div id="container">
+  <div id="target"></div>
+</div>
+<script>
+assert_inherited('color-interpolation-filters', 'auto', 'srgb'); // SVG 1.1 had linearrgb as initial value.
```

Comment:
I think https://github.com/w3c/csswg-drafts/issues/3131 shows that this was an inadvertent change, and should be reverted. This line in the test would then need to be altered, and browsers would pass again :)